.github/unittest/install_dependencies_nightly.sh

```diff
@@ -16,10 +16,8 @@ pip uninstall --yes tensordict
 
 cd ..
 
-git clone https://github.com/pytorch/tensordict.git
-cd tensordict
-python setup.py develop
-cd ..
-git clone https://github.com/pytorch/rl.git
-cd rl
-python setup.py develop
+# install cmake
+conda install anaconda::cmake -y
```

Comment:
swap with apt-get or anything else if needed

Reply:
Thanks! can you have a look at the windows error? is there a special way to install on windows?